src/popup/components/InputField.vue

```diff
@@ -143,6 +150,9 @@ export default {
       }
       return !!this.message;
     },
+    availableTextLimit() {
+      return this.value?.length ? this.textLimit - this.value.length : this.textLimit;
```

Comment:
There is a possibility of type mismatching if the `textLimit` prop is not provided. In this case JS will try to subtract the text length from `null`. This is causing any JS error, but can be considered as a bad practice.